site/cypress/utils/utils.js

```diff
@@ -99,7 +99,7 @@ export const navTester = (path) => {
 
   export const downloadButtonTester = (path) => {
     it('checks the download button to verify it exists',() => {
-      cy.visit(`/apis/${path}`)
+      cy.visit(`/apis/${path}/`)
```

Comment:
Please explain the trailing slash `/` updates here and elsewhere - what happens without change(logic in lambda?), what have you tried to resolve it besides this change ?

Reply:
The `/`s needed to be updated like this because of the way that the lambda pulls out the subdomain and puts it in the path for the s3 bucket. Without changing this here I was getting errors and a strange URI in the links but this small change makes everything work correctly in the new URL

Reply:
So can't we put a slash when it's missing at the end of URL in respective lambdas before custom logic?

Reply:
We could but it seemed like extra work in there that was unnecessary. Currently lambda is taking the subdomain and adding it in the path as the entrypoint. There's nothing defined to handle the end of any particular URI so there's more logic that would need to thought-through and added to get that piece also working on each call. Since every endpoint needs to be added with or without a slash in our site source code anyway, the simpler solution seems to me to just include it on the paths when we define them instead of adding another manipulation of the URI in the lambda. 

That was my thinking at least, if it's more correct/better to define that logic in the lambda and revert these let me know and I'll do that 👍